source/manual/data-gov-uk-operations.html.md

```diff
@@ -24,18 +24,24 @@ This will populate a new index and rotate the `dataset-staging` alias to point t
 
 ## Sync from [CKAN]
 
-This is done automatically using Sidekiq Scheduler. You can run manually to populate your local DB.
+### Perform a full re-sync
+
+The sync is normally done automatically using Sidekiq Scheduler. There may be times when you need to throw away the existing Postgres database, sync all datasets from CKAN and reindex.
+
+This will not make any changes to the content on Find until the reindex has completed and the Elastic index is updated.  This will affect data served on Publish, however this service is not currently available to users. 
 
 ```
+cf ssh publish-data-beta-staging
+
 ## make sure your local DB is empty
```

Comment:
We should not use the phrase "local DB" if we're now talking about running this in prod/staging

Reply:
Yes, I've changed this